app/routes.py

```diff
@@ -1,2 +1,92 @@
-from flask import Blueprint
+import re
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.Planet import Planet
 
+planet_bp = Blueprint("planet", __name__, url_prefix="/planets")
+
+#Validate planet data
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        return abort(make_response({"message": f"Planet {planet_id} is invalid"}, 400))
+
+    planet = Planet.query.get(planet_id)
+    
+    if not planet:       
+        return abort(make_response({"message": f"Planet {planet_id} is not found"}, 404))
+    return planet
```

Comment:
We could move this out of the routes file into its own helper function file, so that the routes are cleaner